frontends/infinite-corridor/src/scss/combined.scss

```diff
@@ -1,5 +1,3 @@
-@use "material-components-web/dist/material-components-web";
```

Comment:
We aren't using this currently. (MUI instead). It's a holdover from very early infinite-corridor homepage.